src/test/java/com/insightfullogic/honest_profiler/ports/sources/LocalMachineSourceTest.java

```diff
@@ -21,7 +21,7 @@
 
 
             it.should("detect local machines", expect -> {
-                AgentRunner.run("InfiniteExample", runner -> {
+                AgentRunner.run("InfiniteExample", "interval=100", runner -> {
```

Comment:
Could `"interval=100"` not be put into a constant if it being referred to in multiple tests?